docs/user/README.md

```diff
@@ -16,9 +16,8 @@ The following commands are for Ubuntu Server. However, equivalent commands shoul
 1. Install [cURL](https://curl.se/) as the *root* user: `pkg install curl`
 
```

Comment:
Very minor and not related to this PR but while reading this I found a small spelling error on line 222. It might as well be fixed now. Should be `add significant overhead` instead.

Reply:
Fixed.